src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -315,10 +329,19 @@ void GcsPlacementGroupManager::RemovePlacementGroup(
     on_placement_group_removed(Status::OK());
     return;
   }
-  auto placement_group = placement_group_it->second;
+  auto placement_group = std::move(placement_group_it->second);
   registered_placement_groups_.erase(placement_group_it);
   placement_group_to_create_callbacks_.erase(placement_group_id);
 
+  // Remove placement group from `named_placement_groups_` if its name is not empty.
+  if (!placement_group->GetName().empty()) {
```

Comment:
Can you add a unit test that tests this code path?

Reply:
Added a ut `TestRemoveNamedPlacementGroup` in gcs_placement_group_manager_test.cc